Sindri-Labs/forge-sample-data
# dependency patch

~~Tiny change to dependencies in a Halo2 circuit to account for `time-macro` package update incompatibility with our prover.~~

Changing Clap package version in Halo2 circuits to account for compatibility with rust version 1.69.

## Testing Instructions

For all of the following, upload to `forge.sindri.app` frontend and compile. Ensure circuit status changes to "Ready"
- [x] `halo2/axiom-v0.3.0/axiom_header_goerli.tar.gz`
- [x] `halo2/axiom-v0.3.0/float_radius.tar.gz`
- [x] `halo2/axiom-v0.3.0/quadratic_circuit.tar.gz`
- [x] `halo2/axiom-v0.2.2/storage_proof.tar.gz`
